enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/MultiRealmAuthManager.java

```diff
@@ -187,6 +188,7 @@ public void shutdown() throws Throwable
                 ((RealmLifecycle) realm).shutdown();
             }
         }
+        securityLog.shutdown();
```

Comment:
I noted that `EnterpriseAuthManagerFactory` has a piece of defensive code that can either accept an externally injected log or create a new one. 
`private SecurityLog getSecurityLog( Log allegedSecurityLog )`

If we get the log injected we should be careful about how we manage its life cycle.